docs/start/introduction.md

```diff
@@ -0,0 +1,84 @@
+laboratories have different providers and different types of communication. To automate, for example, a set of devices 
+to operate a process automatically, it is necessary:
+
+* Define a programming coding language;
+
+* Search with providers for ways to access the device using the chosen language;
+
+* Develop the communication system using computer signal inputs like serial and Ethernet;
+
+* Integrate all devices into a single system, where it is possible to start building protocols and instructions.
+
+In addition to all the complexity involved in the steps presented, a factor that can be stressful is if there is a need 
+to change a device during the construction or operation of the platform. This change will require rework to build the 
+new device system, especially if it comes from a different supplier (with other communication, functions, and methods).
+
+Flowchem connects with each device, ensuring easy access to all available commands through a web interface. This means 
+that the user will not have to limit themselves to using a specific program language, as the connection to the devices 
+can be easily accessed with a web data request package, which is present in most program languages (i.e., python, java, 
+c++, HTML, and others).
+
+Flowchem also has a high level of abstraction concerning devices. This abstraction occurs through the use of class, 
+which represents components of a device, to build equipment. To explain better, look at the table below.
+
+| Class of the system | Description        | Main function            | 
+|---------------------|--------------------|--------------------------|
+| Azura Compact Pump  | Device             | Communication Configure  |
+| HPLC pump           | Component Specific | Specific access commands |
+| Pump                | Component Base     | Base access commands     |
+| Flowchem Component  | Class Base         | Base Constructor         |
+
+When constructing a class to represent a device, as in our case, ***Azura Compact Pump*** inherits the attributes and 
+methods of the class ***HPLC Pump***. This ***HPLC Pump*** class presents the specific standard commands for this pump 
+category. In turn, this class inherits the attributes and methods of the class ***Pump***. This class has the most 
+generic commands in all pumps (activate, infuse). This class still features the main class with attributes and methods 
+present on all devices. This level of abstraction allows the user to switch from one device to another very quickly, 
+as long as both devices have the same function (inherit the commands from ***HPLC Pump***, for example). It doesn't 
+matter if they come from suppliers and have different communication protocols. This device exchange will occur without
+changing anything in the already implemented code structure (a program related to the platform's automation). This 
+change will only occur in the configuration file, which is easily editable.
+
```

Comment:
When constructing a class to represent a device, such as the Azura Compact Pump, it inherits the attributes and methods of the HPLC Pump class. This HPLC Pump class provides the standard commands specific to its pump category. In turn, the HPLC Pump class inherits the attributes and methods of the more generic Pump class, which includes basic commands common to all pumps (like activate and infuse). Furthermore, this Pump class inherits from a main class that contains attributes and methods applicable to all devices.

This level of abstraction allows users to switch between devices quickly, as long as the devices share the same function (i.e., inherit commands from the HPLC Pump class, for example). It doesn't matter if the devices come from different suppliers or have different communication protocols. This device exchange can be done without altering the existing code structure related to the platform's automation. The change is only needed in the configuration file, which is easily editable.